tests/media_nofilename_test.go

```diff
@@ -52,8 +53,19 @@ func TestMediaWithoutFileName(t *testing.T) {
 			mxc := alice.UploadContent(t, file, fileName, contentType)
 			must.NotEqualStr(t, mxc, "", "did not return an MXC URI")
 			must.StartWithStr(t, mxc, "mxc://", "returned invalid MXC URI")
+
 			b, ct := alice.DownloadContent(t, mxc)
-			must.EqualStr(t, ct, contentType, "wrong Content-Type returned")
+
+			// Check the Content-Type response header.
+			// Homeservers are free to add other directives. All we want to check is the mimetype.
+			mimeType := strings.Split(ct, ";")[0]
```

Comment:
Please add `NOTSPEC:` and explain the ambiguities that @turt2live pointed out.

Reply:
Oh ok. Should we add this wording to `ONBOARDING.md` or similar to standardise it?